libraries/core-react/src/components/Input/Input.tsx

```diff
@@ -101,18 +84,22 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
   { variant = 'default', disabled = false, type = 'text', ...other },
   ref,
 ) {
-  const inputVariant = inputVariants[variant]
+  const actualVariant = variant === 'default' ? 'input' : variant
+  const inputVariant = tokens[actualVariant]
```

Comment:
I am getting a type error here? 
![image](https://user-images.githubusercontent.com/1070981/125058888-fedeab80-e0aa-11eb-9d39-22bff55abcf1.png)

Reply:
Lol me too! See last build for lint error